skaffold.yaml

```diff
@@ -9,23 +9,36 @@ build:
   - linux/arm64/v8
```

Comment:
When we update skaffold.yaml we should change the first line to apiVersion `skaffold/v3`